templates/search/blocks/search_filters.html

```diff
@@ -74,7 +74,9 @@ <h4 class="search-filters__accordion-section-heading">Date</h4>
                             <legend>
                                 <h4 class="search-filters__accordion-section-heading" id="record_opening_date">Record opening date</h4>
                             </legend>
+                            <!--
                             <p id="search_filters__opening-date-help-text">Helper text here (needed for when YYYY MM DD are eventually made separate fields)</p>
```

Comment:
Hey @JohnHeeryTNA, 

from the tickets description I would have thought just removing this completely if the need for the text has been removed completely, but I'll approve this and leave for you to decide. 

Thanks - Will.

Reply:
as discussed, we may revisit this in the future, so commented out for now.